build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
     dependencies {
         classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
         classpath('io.spring.gradle:dependency-management-plugin:0.5.2.RELEASE')
+        classpath 'org.springframework:springloaded:1.2.0.RELEASE'
```

Comment:
Bardzo fajny gadżet, natychmiast podmienia wszystkie html/css/js i klasy javowe. Dodatkowo według strony projektu działa z każdym językiem na JVM. Nie wiem kiedy to może być niewygodne ;)